.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Nitpick: You probably need to add `/*` at the end of `.vscode` ignore as such files shouldn't exist in a repo. 😉